CHANGELOG.md

```diff
@@ -4,10 +4,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.0.0-beta.3]
 
 ### Added
-* The ability to set a custom rest frequency for saving subimages. ([#1653](https://github.com/CARTAvis/carta-frontend/issues/1653)).
+* Added cursor information to the histogram widget ([#1762](https://github.com/CARTAvis/carta-frontend/issues/1762)).
+* The ability to load files with LEL (lattice expression language) expressions ([#1264](https://github.com/CARTAvis/carta-frontend/issues/1264)).
+* The ability to set a custom rest frequency for saving subimages ([#1653](https://github.com/CARTAvis/carta-frontend/issues/1653)).
+* The ability to load complex images with a dropdown menu in the file browser dialog ([#1492](https://github.com/CARTAvis/carta-frontend/issues/1492)).
+### Changed
+* Optimisation to the region list widget with a large number of regions ([#1252](https://github.com/CARTAvis/carta-frontend/issues/1252)).
+* Optimisation to loading images with a million channels ([#1774](https://github.com/CARTAvis/carta-frontend/issues/1774)).
```

Comment:
Absolutely minor, and probably irrelevant. But I looked across the changelog and it looks more American English than British. I think this should be consistent. I suggest you change Optimisation to Optimization, just because with a quick scan I found that everything is American English-like, for instance, color instead of colour.

Reply:
Good point. This is changed.